source/services/relay.md

```diff
@@ -67,13 +68,14 @@ More info about the signature values in appendix F of the [Ethereum Yellow Paper
 **Note:** Atomic operation, many values of s are invalid which are generated by the server.
 
 Clients should verify the server’s response with the following process:
-1. Reconstruct sender address from `signature`
-1. Reconstruct Safe address with sender address and `nonce = 0`
-1. Verify that `Safe` checksum and reconstructed Safe addresses checksum are matching
-1. Verify that `signature` is correct for hash of `tx`
-1. Verify that `tx` has valid bytecode (postponed until mainnet release).
-1. If all checks pass, then transaction and Safe address are valid and user can transfer at least `payment` amount of ETH to the Safe address.
-1. Otherwise, the response has error or it is compromised, and it should not be used any further.
+1. Verify that the `s` that you provided in the request matches the `s` returned in the response.
+1. Verify that `tx.data` matches the bytecode of the Gnosis Safe Proxy contract with the correct owners and threshold. The `value` and `nonce` of the transaction should be zero.
+1. Hash the transaction object (`tx`) and recover the account with the provided `signature`. The resulting address should match the `funder`.
```

Comment:
Actually we don't provide the address of the account that will deploy the contract (as @Uxio0 pointes out), but we still need to derive it here for the next step.

But since checking the Safe address in the end is sufficient we don't need this account in the server response.

Reply:
I understand that you can get the `deployer` from the tx, and then the safe address from the `deployer` + `nonce 0`, but I would let them as a response from the service as they don't harm

Reply:
yeah, I wasn't saying we should remove it. It was just missing in the docs referenced here, maybe we should add `deployer` ;) (see swagger)

Reply:
Hehe, you are right 😅